src/components/proximities/proximities-section.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faUtensils,
+  faTree,
+  faSchool,
+  faUniversity,
+  faHospital,
+  faStore,
+  IconDefinition,
+} from '@fortawesome/free-solid-svg-icons';
+import './proximities-section.css';
+import { H2 } from '../../styles/typography/h2';
+import { Body2 } from '../../styles/typography/body-2';
+import { colors } from '../../styles/colors';
+import { H3 } from '../../styles/typography/h3';
+import { Separator } from '../separator/separator';
+import { ProximitiesMap } from './proximities-map';
+
+const proximityIcons: { [key: string]: IconDefinition } = {
+  WellRatedRestaurants: faUtensils,
+  GreenAreas: faTree,
+  Schools: faSchool,
+  Colleges: faUniversity,
+  Hospitals: faHospital,
+  Markets: faStore,
+};
+
+export interface Proximity {
+  type: keyof typeof proximityIcons;
+  description: string;
+}
+
+interface ProximitiesSectionProps {
+  proximities: Proximity[];
+  latitude: number;
+  longitude: number;
+}
+
+export const ProximitiesSection = ({ proximities, latitude, longitude }: ProximitiesSectionProps) => {
+  return (
+    <section className="proximities-section">
+      <div className="proximity-text">
+        <H2 color="NeutralXdark"> Proximidades </H2>
```

Comment:
nao sei se os espaços eram propositais, mas acredito que o html vai ta ignorando eles. Se vc quiser que eles existam, vc pode usar a unicode deles `\u0020`
 
```suggestion
        <H2 color="NeutralXdark">Proximidades</H2>
```